app/presenters/tree_builder_ops_features.rb

```diff
@@ -0,0 +1,249 @@
+class TreeBuilderOpsFeatures < TreeBuilder
+  include CompressedIds
+
+  has_kids_for Hash, [:x_get_tree_hash_kids]
+
+  attr_reader :role, :features, :editable
+
+  def initialize(name, type, sandbox, build, role:, features:, editable: false)
+    @role     = role
+    @editable = editable
+    @features = features
```

Comment:
Why not skip `features` and do `@feature = @role.feature_identifiers.sort` instead?